docs/loading/RoutineLoad.md

```diff
@@ -268,6 +268,128 @@ After submitting the load job, you can execute the [SHOW ROUTINE LOAD](../sql-re
+
+```SQL
+CREATE TABLE sensor.sensor_log ( 
+    `id` bigint NOT NULL COMMENT "sensor id",
+    `name` varchar(26) NOT NULL COMMENT "sensor name", 
+    `checked` boolean NOT NULL COMMENT "checked", 
+    `data` double NULL COMMENT "sensor data", 
+    `sensor_type` varchar(26) NOT NULL COMMENT "sensor type"
+) 
+ENGINE=OLAP 
+PRIMARY KEY (id) 
+DISTRIBUTED BY HASH(`id`) BUCKETS 5; 
+```
+
+#### Submit a Routine Load job
+
+Execute the following statement to submit a Routine Load job named `sensor_log_load_job` to consume the Avro messages in the Kafka topic `topic_0` and load the data into the table `sensor_log` in the database `sensor`. The load job consumes the messages from the initial offset in the specified partitions of the topic.
+
+```SQL
+CREATE ROUTINE LOAD example_db.sensor_log_load_job ON sensor_log1  
+PROPERTIES  
+(  
+    "format" ="avro"  
+)  
+FROM KAFKA  
+(  
+    "kafka_broker_list" ="<kafka_broker1_ip>:<kafka_broker1_port>,<kafka_broker2_ip>:<kafka_broker2_port>,...",
+    "confluent.schema.registry.url"="http://172.xx.xxx.xxx:8081",  
+    "kafka_topic"="topic_0",  
+    "kafka_partitions"="0,1,2,3,4,5",  
+    "property.kafka_default_offsets"="OFFSET_BEGINNING"  
+);
+```
+
+- Data Format
+
+  You need to specify `"format"="avro"` in the clause `PROPERTIES` to define that the data format is Avro.
+
+- Schema Registry `confluent.schema.registry.url`: the URL of the Schema Registry where the Avro schema is registered. StarRocks retrieves the Avro schema by using this URL. The format is as follows:
+
+    ```Plaintext
+    confluent.schema.registry.url = http[s]://[<schema-registry-api-key>:<schema-registry-api-secret>@]<hostname|ip address>[:<port>]
+    ```
+
+- Data mapping and transformation
+
+  To specify the mapping and transformation relationship between the Avro-format data and the StarRocks table, you need to specify the paramter `COLUMNS` and property `jsonpaths`. The order of fields specified in the `COLUMNS` parameter must match that of the fields in the property `jsonpaths`, and the names of fields must match these of the StarRocks table. The property `jsonpaths` is used to extract the required fields from the Avro data. These fields are then named by the property `COLUMNS`.
+
+  For more information about data transformation, see [Transform data at loading](https://docs.starrocks.io/en-us/latest/loading/Etl_in_loading).
+
+  > NOTE
+  >
+  > You do not need to specify the `COLUMNS` parameter if the names and number of the fields in the Avro record completely match those of columns in the StarRocks table.
+
+After submitting the load job, you can execute the [SHOW ROUTINE LOAD](../sql-reference/sql-statements/data-manipulation/SHOW%20ROUTINE%20LOAD.md) statement to check the status of the load job.
+
+#### Data types mapping
+
+StarRocks supports loading Avro data of all types. When Avro data is loaded into StarRocks, the data type mapping is as follows:
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/32645c51-9982-471a-9f3b-a7e9824a4b1a)
I think this should mean -> The data type mapping between the Avro data fields you want to load and the StarRocks table columns is as follows: